Numpy int64 - known issues - docs changed fixes #8527 (#8537)
astropy/astropy: docs/known_issues.rst

but only when running in the IPython console.  Colors are not currently
supported in the basic Python command-line interpreter on Windows.

> numpy.int64 does not decompose input

Reviewer: ```suggestion
numpy.int64 does not decompose input ``Quantity`` objects
```

> ------------------------------------

Reviewer: ```suggestion
---------------------------------------------------------
```

::

    >>> np.int64((15 * u.km) / (15 * u.imperial.foot))
    1
    >>> np.int((15 * u.km) / (15 * u.imperial.foot))
    3280

>     >>> int is np.int

Reviewer: I would remove this line and the example below in favor of an example above showing what happens with `int((15 * u.km) / (15 * u.imperial.foot))`.

    True
    >>> np.int_ is np.int64
    True

> Python's int or np.int goes through ``__index__`` while np.int64 or `np.int_` does not go through ``__index__``

Reviewer 2: Looks good modulo the missed links. Try `numpy.int64` and `numpy.int_`

Also, full stop at the end of this sentence, and break it to <80 characters.

> Python's int or np.int goes through ``__index__`` while np.int64 or `np.int_` does not go through ``__index__``

Reviewer: Here is a suggested replacement:
```suggestion
Python's ``int()`` or `numpy.int` goes through ``__index__`` while `numpy.int64` 
or `numpy.int_` does not go through ``__index__``
```

> Hence int(...) is to be used if an integer is needed.

Reviewer: ```suggestion
Hence `int(...)` should be used if an integer is needed from a dimensionless ``Quantity``.
```
